apps/condo/domains/acquiring/access/RegisterMultiPaymentForOneReceiptService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by `createservice acquiring.RegisterMultiPaymentForOneReceiptService`
+ */
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canRegisterMultiPaymentForOneReceipt ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return user.isAdmin || user.type === RESIDENT
```

Comment:
It's not for Residents! It's for anyone who have access to billing receipt! and integration context.

1 case: resident with right service consumer
2 case: organization admin with manage billing and acquiring context access